k9mail/src/test/java/com/fsck/k9/helper/AppUpdaterTest.kt

```diff
@@ -125,10 +141,17 @@ class AppUpdaterTest {
     }
 
     private fun verifyV3111Update() {
-        storage.getBoolean("pEpUsePassphraseForNewKeys", BuildConfig.USE_PASSPHRASE_FOR_NEW_KEYS)
+        //storage.getBoolean("pEpUsePassphraseForNewKeys", BuildConfig.USE_PASSPHRASE_FOR_NEW_KEYS)
+        storageEditor.remove("pEpUsePassphraseForNewKeys")
```

Comment:
I don't understand how the verify works for 3111, I understand the 3114, but the 3111 seems to be just removing and putting a new property, without testing it.

Reply:
ok found it, I didn't at first as it was not modified, should have checked in IDE